content/docs/user-guide/gems/development/creating/_index.md

```diff
@@ -14,13 +14,13 @@ Although you can create a Gem manually by creating all of the files yourself, it
 o3de create-gem -gp <path to create gem at>
 ```
 
-This will create your Gem with a standard file structure and CMake files created from templates.
+This will create your Gem with a standard file structure and CMake files created from [templates](https://github.com/o3de/o3de/tree/development/Templates).
 
```

Comment:
Suggestion - Navigate users to the CLI Reference page so they can quickly find out how to run the `o3de` command for their platform. This is helpful for users who aren't familiar with the `o3de` tool. The linked _CLI Reference_ page is currently a placeholder page, but PR https://github.com/o3de/o3de.org/pull/763 will replace that.

```suggestion

For more information on how to use the `o3de` tool, refer to [Project Configuration CLI Reference](/docs/user-guide/project-config/cli-reference/)

```

Reply:
That makes sense, if we can point to another documentation page it keeps the user in the same frame of mind at looking at the o3de.org documentation pages.